django/contrib/admin/utils.py

```diff
@@ -362,6 +368,10 @@ def label_for_field(name, model, model_admin=None, return_attr=False):
                     label = pretty_name(attr.__name__)
             else:
                 label = pretty_name(name)
+    except FieldIsAForeignKeyColumnName:
+        label = pretty_name(name)
```

Comment:
Is this branch tested? (i.e. if I change this to `label='xxx'` do any tests fail?)